react/features/base/conference/middleware.any.ts

```diff
@@ -339,12 +339,44 @@ function _conferenceJoined({ dispatch, getState }: IStore, next: Function, actio
  * @private
  * @returns {Object} The value returned by {@code next(action)}.
  */
-function _connectionEstablished({ dispatch }: IStore, next: Function, action: AnyAction) {
+async function _connectionEstablished({ dispatch, getState }: IStore, next: Function, action: AnyAction) {
     const result = next(action);
 
     // FIXME: Workaround for the web version. Currently, the creation of the
     // conference is handled by /conference.js.
-    typeof APP === 'undefined' && dispatch(createConference());
+    if (typeof APP === 'undefined') {
+        dispatch(createConference());
+
+        return result;
+    }
+
+    // TODO keep this here till we move tracks and conference management from
+    // conference.js to react.
+    const state = getState();
+    let localTracks = getLocalTracks(state['features/base/tracks']);
+
+    // Do not signal audio/video tracks if the user joins muted.
+    for (const track of localTracks) {
```

Comment:
Do we really need this. Aren't we filtering all those tracks in conference.js just before the dispatching `connect`?

Reply:
But if you look at the old code, we were doing it already in react/features/prejoin/actions.web.ts, so I just moved it.

Reply:
I will revert my latest commit as it is not correct and will re-think it again.

Reply:
This code is executed if you use prejoin screen so we need that filtering as user may have adjusted the tracks mute/unmute state. I have a solution for prejoin and non prejoin thanks to @jallamsetty1 :) 
Will test it and push it in a bit.